packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -173,6 +181,15 @@ export class SageMakerCreateTransformJob extends sfn.TaskStateBase {
     };
   }
 
+  private renderModelClientOptions(config: ModelClientOptions): { [key: string]: any } {
+    return {
+      ModelClientConfig: {
+        InvocationsMaxRetries: config.invocationsMaxRetries ?? 0,
```

Comment:
i don't think we should explicitly set retries to zero. it feels like a bad default, we should let the service defaults take over (which I think is 3?)

Reply:
0 is actually the default for retries. Typically, after the first try fails, the job fails. I just ran into it today and so had a chance to confirm it :).

Reply:
How do you recommend we let the service defaults take over? The reason I added those was to avoid the case when one of `invocationsMaxRetries` or `invocationsTimeout` was set, but not both. That is the only case when it would get to this function. Maybe simply replace this with: `InvocationsMaxRetries: config.invocationsMaxRetries?` would do it?

Reply:
Updated these to `InvocationsMaxRetries: config.invocationsMaxRetries?` and similar for the timeout. So, CDK doesn't set the default values but lets the service-level defaults take over.

Reply:
would it make sense to set it after some basic validation? - i.e. it exists, and is greater than `0`? - otherwise, I think it should be fine if the service defaults are 0. wdyt?

Reply:
Agree, it should also be < 3, so I can add validations for both > 0 and < 3 to it later today.